examples/with-vue-thirdpartyemailpassword/src/views/AuthView.vue

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import SuperTokensReactComponent from "../components/Supertokens";
+
+export default {
+    beforeMount() {
+        this.render();
+    },
+    beforeUpdate() {
+        this.render();
+    },
+    mounted() {
+        this.render();
+    },
```

Comment:
Hey, I am not very familiar with Vue but do you need to load the react component in `beforeMount` and `beforeUpdate`.  We've updated the [angular example](https://github.com/supertokens/supertokens-auth-react/blob/0.23/examples/with-angular-thirdpartyemailpassword/src/app/auth/auth.component.ts#L18) also to show this, but we would like to initialize the react component only once after your the auth view component finishes initializing.

Reply:
Yeah, the `beforeMount` and `beforeUpdate` methods aren't needed. Have updated the PR to reflect that change.